framework/src/problems/ReferenceResidualProblem.C

```diff
@@ -530,7 +544,11 @@ ReferenceResidualProblem::checkConvergenceIndividVars(const Real fnorm,
   {
     for (unsigned int i = 0; i < _group_resid.size(); ++i)
       convergedRelative &=
-          ((_group_resid[i] < _group_ref_resid[i] * rtol) || (_group_resid[i] < abstol));
+          (_group_resid[i] < _group_ref_resid[i] * rtol || _group_resid[i] < abstol ||
+           (_group_ref_resid[i] == 0.0 &&
+            ((_zero_ref_type == ZeroReferenceType::ZERO_TOLERANCE && _group_resid[i] == 0.0) ||
+             (_zero_ref_type == ZeroReferenceType::RELATIVE_TOLERANCE &&
+              _group_resid[i] <= rtol))));
```

Comment:
Isn't this an absolute tolerance?

Reply:
It just seems like this comparison is not relative to anything, so I don't understand the use of a "relative" tolerance

Reply:
Relative to zero.
```
zero_ref_res.addDocumentation(
      "relative_tolerance",
      "Solve is treated as converged if the residual is below the relative tolerance");
```

Reply:
Haha but isn't that just an absolute tolerance? Sorry if I'm being dense. I don't understand yet what this new capability gives that isn't achievable with an absolute tolerance

Reply:
Maybe you don't need to change the name of the enumerator, but I would appreciate a comment above the comparison about why you are comparing against a relative tolerance in an absolute way. Then future developers coming across the code won't have the same questions I'm peppering you with now.

With respect to the default, have you tried making it your new option and seeing if anything breaks in CIVET?

Reply:
Sounds good. I really like the comment!